homeassistant/components/nibe_heatpump/sensor.py

```diff
@@ -29,6 +29,78 @@
 
 from . import DOMAIN, CoilEntity, Coordinator
 
+UNIT_DESCRIPTIONS = {
+    TEMP_CELSIUS: SensorEntityDescription(
```

Comment:
Do you prefer not using HA constants here and write it out explicitly? "°C"

Reply:
@MartinHjelmare

Reply:
The Home Assistant constants should only be used in the Home Assistant side of the map, ie not for the map keys or description keys.